influxql/ast.go

```diff
@@ -372,7 +372,7 @@ func (s *CreateUserStatement) String() string {
 	_, _ = buf.WriteString("CREATE USER ")
 	_, _ = buf.WriteString(s.Name)
 	_, _ = buf.WriteString(" WITH PASSWORD ")
-	_, _ = buf.WriteString(s.Password)
```

Comment:
@jwilder I had the same though at first but then decided it would be better for `String()` to redact sensitive info, like passwords.  We can add a `StringUnsafe()` to those few statements, for internal message passing code to use when the string needs to be re-parsed.  Thoughts?